framework/web/JsonResponseFormatter.php

```diff
@@ -38,6 +38,11 @@
 class JsonResponseFormatter extends Component implements ResponseFormatterInterface
 {
     /**
+     * @var string|null the MIME type used in the `Content-Type` header of the response.
+     * When equals `null` default MIME type will be used based on the `useJsonp` property.
+     */
+    public $mimeType = null;
```

Comment:
I'd name it the same way as in `XmlResponseFormatter`. Naming it differently would cause huge usability issue while the name `contentType` is OK.